shell/platform/linux/fl_text_input_plugin.cc

```diff
@@ -58,6 +59,9 @@ struct FlTextInputPluginPrivate {
   // Send newline when multi-line and enter is pressed.
   gboolean input_multiline;
 
+  // The input method is not shown when the input type is "none".
+  gboolean input_none;
```

Comment:
Can you replace `input_multiline`/`input_none` with a enum `input_type` - we can't be both none and multiline.